app/models/addressLookup/AddressModel.scala

```diff
@@ -35,3 +63,29 @@ object AddressModel {
   implicit val fmt: Format[AddressModel] = Json.format[AddressModel]
 
 }
+
+object AmendAddressModel {
+
+  implicit val fmt: Format[AmendAddressModel] = Json.format[AmendAddressModel]
+}
+
+case class AmendAddressModel(line1: String,
+                             line2: Option[String],
+                             line3: Option[String],
+                             town: String,
+                             postcode: String,
+                             country: String) {
+
+  def toConfirmableAddress(implicit messages: Messages): AddressModel =
+    AddressModel(
+      Seq(line1) ++ line2.map(line2 => line2).toList ++ line3.map(line3 => line3).toSeq ++ Seq(town),
```

Comment:
the .maps are not needed, also is .toList needed for any special reason or can it be .toSeq, for consistency?